cmd/podman/system/version.go

```diff
@@ -41,31 +42,38 @@ func version(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
-	switch {
-	case parse.MatchesJSONFormat(versionFormat):
+	if parse.MatchesJSONFormat(versionFormat) {
 		s, err := json.MarshalToString(versions)
 		if err != nil {
 			return err
 		}
-		_, err = io.WriteString(os.Stdout, s+"\n")
-		return err
-	case cmd.Flag("format").Changed:
-		out := formats.StdoutTemplate{Output: versions, Template: versionFormat}
-		err := out.Out()
+		fmt.Println(s)
+		return nil
+	}
+
+	w := tabwriter.NewWriter(os.Stdout, 0, 0, 2, ' ', 0)
```

Comment:
Unless they're on top of each other, I really dislike short variable names, especially so one char ones.  Searching for them is just too painful when updating/fixining two years from now.